mantaray/backend.py

```diff
@@ -135,6 +135,8 @@ def __init__(self, server_config: config.ServerConfig, *, verbose: bool):
         self.event_queue: queue.Queue[IrcEvent] = queue.Queue()
         self._thread: threading.Thread | None = None
 
+        self.nickmask = ""
```

Comment:
I want mypy to warn me, if I forget that the mask might not be available:

```suggestion
        self.nickmask: str | None = None
```